### DISCUSSION TITLE
How I can change styles from range slider?

### DISCUSSION THREAD

**Discussion by webcreative24** [breezefront/community#89] [1. Help] [answered]
Hello. I've installed Smile elastic suite additional extension for Breeze compatibility. How I can change the styles of the price range slider, ie: line color, and range buttons:
![image](https://user-images.githubusercontent.com/13699502/230781461-a114fc92-c01b-4bd9-9ba6-140597e9ded8.png)
I cannot find any related styles via browser dev tools. can you please help? Thanks in advance

**Comment by vovayatsyuk** [comment]
Here are the styles that we use for this element: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/web/css/base/_reset.less#L258-L307

To see how they are styled in dev tools, I recommend enabling "Show user agent shadow DOM" option in Chrome dev toolbar settings:

![image](https://user-images.githubusercontent.com/306080/230845192-27934cc5-63c7-478c-9030-eef1fd05bbd2.png)

Then you'll be able to inspect these elements:

![image](https://user-images.githubusercontent.com/306080/230845375-29a2dfad-5f9b-47d3-9627-50f627971e91.png)

**Reply by webcreative24** [reply]
Thank you!

**Comment by webcreative24** [comment]
Hi, @vovayatsyuk The price slider looks weird in the Firefox browser. can you please help me how to fix it? 
![image](https://user-images.githubusercontent.com/13699502/233413116-31e47cc5-967f-474b-8eb3-e99d6cb2b549.png)
I can't change the range buttons position and slider line color in the Firefox

**Reply by vovayatsyuk** [reply, marked as answer]
Just fixed Firefox styles: https://github.com/breezefront/module-breeze/commit/7292918e992d50fab9fa44973bfce8260a1fa623
I've recently added docs about the range slider, have a look: https://breezefront.com/docs/range-slider#styles
